README.md

```diff
@@ -31,6 +31,9 @@ or a traditional socket endpoint.
 ## DCC (Data Center Component)
 The abstract class DCC represents an application in a data-center. It is potentially the most important and complex abstraction of liota. It provides flexibility to developers for choosing the data-center components they need and using API’s provided by liota. With help of this abstraction developers may build custom solutions. The abstract class states basic methods and encapsulates them into unified common API’s required to send data to various DCC’s. Graphite and Project Ice are currently the data-center components supported with AWS, BlueMix and ThingWorx to come soon. New DCC’s can easily be integrated in the abstraction.
 
+## Transports
```

Comment:
Mention in the README.md that Gateway object should be created before creating DCC_comms object. And also how to set the Edge system identity and DCC identity should be made part of main REDME.md